packages/pieces/http/src/lib/actions/send-http-request-action.ts

```diff
@@ -38,8 +53,23 @@ export const httpSendRequestAction = createAction({
       headers: headers as HttpHeaders,
       queryParams: queryParams as QueryParams,
       body,
+      timeout: timeout ? timeout * 1000 : 0,
     };
 
-    return await httpClient.sendRequest(request);
+    try {
+        const response = await httpClient.sendRequest(request);
+        return {
+            success: response.status && response.status == 200,
```

Comment:
2xx is considered success, for example some services might return 201 which means okay.

This will fail in that case.

Reply:
Taken care of.